core/schema/src/main/resources/org/hawkular/schema/updates/schema-0.27.0.groovy

```diff
@@ -0,0 +1,263 @@
+) WITH CLUSTERING ORDER BY (time DESC)
+"""
+}
+schemaChange {
+  version '8.0'
+  author 'burmanm'
+  tags '0.27.x'
+  cql """
+CREATE TABLE data_temp_5 (
+    tenant_id text,
+    type tinyint,
+    metric text,
+    dpart bigint,
+    time timestamp,,
+    n_value double,
+    availability blob,
+    l_value bigint,
+    tags map<text,text>,
+    PRIMARY KEY ((tenant_id, type, metric, dpart), time)
+) WITH CLUSTERING ORDER BY (time DESC)
+"""
+}
+schemaChange {
+  version '8.0'
+  author 'burmanm'
+  tags '0.27.x'
+  cql """
+CREATE TABLE data_temp_6 (
+    tenant_id text,
+    type tinyint,
+    metric text,
+    dpart bigint,
+    time timestamp,,
+    n_value double,
+    availability blob,
+    l_value bigint,
+    tags map<text,text>,
+    PRIMARY KEY ((tenant_id, type, metric, dpart), time)
+) WITH CLUSTERING ORDER BY (time DESC)
+"""
+}
+schemaChange {
+  version '8.0'
+  author 'burmanm'
+  tags '0.27.x'
+  cql """
+CREATE TABLE data_temp_7 (
+    tenant_id text,
+    type tinyint,
+    metric text,
+    dpart bigint,
+    time timestamp,,
+    n_value double,
+    availability blob,
+    l_value bigint,
+    tags map<text,text>,
+    PRIMARY KEY ((tenant_id, type, metric, dpart), time)
+) WITH CLUSTERING ORDER BY (time DESC)
+"""
+}
+schemaChange {
+  version '8.0'
+  author 'burmanm'
+  tags '0.27.x'
+  cql """
+CREATE TABLE data_temp_8 (
+    tenant_id text,
+    type tinyint,
+    metric text,
+    dpart bigint,
+    time timestamp,,
+    n_value double,
+    availability blob,
+    l_value bigint,
+    tags map<text,text>,
+    PRIMARY KEY ((tenant_id, type, metric, dpart), time)
+) WITH CLUSTERING ORDER BY (time DESC)
+"""
+}
+schemaChange {
+  version '8.0'
+  author 'burmanm'
+  tags '0.27.x'
+  cql """
+CREATE TABLE data_temp_9 (
+    tenant_id text,
+    type tinyint,
+    metric text,
+    dpart bigint,
+    time timestamp,,
+    n_value double,
+    availability blob,
+    l_value bigint,
+    tags map<text,text>,
+    PRIMARY KEY ((tenant_id, type, metric, dpart), time)
+) WITH CLUSTERING ORDER BY (time DESC)
+"""
+}
+
+schemaChange {
+  version '8.0'
+  author 'burmanm'
+  tags '0.27.x'
+  cql """
+CREATE TABLE data_temp_10 (
+    tenant_id text,
+    type tinyint,
+    metric text,
+    dpart bigint,
+    time timestamp,,
+    n_value double,
+    availability blob,
+    l_value bigint,
+    tags map<text,text>,
+    PRIMARY KEY ((tenant_id, type, metric, dpart), time)
+) WITH CLUSTERING ORDER BY (time DESC)
+"""
+}
+
+schemaChange {
+  version '8.0'
+  author 'burmanm'
+  tags '0.27.x'
+  cql """
+CREATE TABLE data_temp_11 (
+    tenant_id text,
+    type tinyint,
+    metric text,
+    dpart bigint,
+    time timestamp,,
+    n_value double,
+    availability blob,
+    l_value bigint,
+    tags map<text,text>,
+    PRIMARY KEY ((tenant_id, type, metric, dpart), time)
+) WITH CLUSTERING ORDER BY (time DESC)
+"""
+}
+
+// Lets hope this will never be compacted.. just to be on the safe-side though..
+schemaChange {
```

Comment:
I would probably stick with STCS especially considering that within a given time bucket TWCS  uses STCS.